mmv1/products/compute/terraform.yaml

```diff
@@ -725,10 +725,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           mig_template_name: "l7-ilb-mig-template"
           hc_name: "l7-ilb-hc"
           mig_name: "l7-ilb-mig1"
-          fw-allow_iap_hc_name: "l7-ilb-fw-allow-iap-hc"
+          fw_allow_iap_hc_name: "l7-ilb-fw-allow-iap-hc"
```

Comment:
Same comment as https://github.com/GoogleCloudPlatform/magic-modules/pull/5105#discussion_r692895380 . We'll have to edit the old example template

Reply:
Done